### DISCUSSION TITLE
Add FreeBSD support to GDevelop 5

### DISCUSSION THREAD

**Discussion by Prismisho** [4ian/GDevelop#2793] [Technical questions about the codebase]
The title says it all, is it possible?

**Comment by arthuro555** [comment]
Electron doesn't support FreeBSD and all unofficial ports that were ever attempted are outdated and deprecated. Without electron support, GDevelop cannot be run on it.

**Reply by 4ian** [reply]
Best bet here is probably to improve the web-app to support more use cases (import external resources notably).
